packages/wallet/frontend/src/components/layouts/AuthLayout.tsx

```diff
@@ -38,8 +38,8 @@ const AuthLayout = ({ image, children }: AuthLayoutProps) => {
             loading="eager"
           />
         </div>
-        <div className="min-h-full flex-1 py-10">
-          <div className="mx-auto flex min-h-full w-full flex-col items-center sm:px-6 lg:px-20 xl:px-24">
+        <div className="min-h-full flex-1">
+          <div className="mx-auto flex h-screen w-full flex-col items-center px-2">
```

Comment:
I added h-screen here because min-h-full added a vertical scroll, and the links for registering and logging in were not shown on screen. Also this currently makes the theme toggle on mobile device overlap the logo, but we will remove the toggle soon.
- padding is removed here because GateHub KYC has already padding